utils/sites.js

```diff
@@ -301,28 +302,73 @@ const updateParticipantData = async (req, res, authObj) => {
             }
         }
 
-        if(updatedData['399159511']) updatedData[`query.firstName`] = dataObj['399159511'].toLowerCase();
-        if(updatedData['996038075']) updatedData[`query.lastName`] = dataObj['996038075'].toLowerCase();
+        // Handle deceased data. participantDeceased === yes && participantDeceasedTimestamp req'd. Derive participantDeceasedNORC === fieldMapping.yes.
+        // Ignore and delete deceased data if participantDeceased === no. Return error for incomplete submission.
+        if (updatedData[fieldMapping.participantDeceased] === fieldMapping.yes && updatedData[fieldMapping.participantDeceasedTimestamp]) {
+            updatedData[fieldMapping.participantDeceasedNORC] = fieldMapping.yes;
+        } else if (updatedData[fieldMapping.participantDeceased] === fieldMapping.no) {
+            delete updatedData[fieldMapping.participantDeceased];
+            delete updatedData[fieldMapping.participantDeceasedTimestamp];
+        } else if (updatedData[fieldMapping.participantDeceased] || updatedData[fieldMapping.participantDeceasedTimestamp]) {
```

Comment:
Unfortunately no because if none of the conditions are true, it isn't part of this update and we just want the data to pass through instead of catching it with `else`